db/migrations/0001_initial.py

```diff
@@ -1,44 +1,58 @@
+            name='Actor',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('first_name', models.CharField(max_length=255)),
+                ('last_name', models.CharField(max_length=255)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='CinemaHall',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('rows', models.IntegerField()),
+                ('seats_in_row', models.IntegerField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Genre',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255, unique=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Movie',
             fields=[
-                (
-                    "id",
-                    models.AutoField(
-                        auto_created=True,
-                        primary_key=True,
-                        serialize=False,
-                        verbose_name="ID",
-                    ),
-                ),
-                ("first_name", models.CharField(max_length=255)),
-                ("last_name", models.CharField(max_length=255)),
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=255)),
+                ('description', models.TextField()),
+                ('actors', models.ManyToManyField(related_name='movies', to='db.Actor')),
+                ('genres', models.ManyToManyField(related_name='movies', to='db.Genre')),
             ],
         ),
         migrations.CreateModel(
-            name="Genre",
+            name='MovieSession',
             fields=[
-                (
-                    "id",
-                    models.AutoField(
-                        auto_created=True,
-                        primary_key=True,
-                        serialize=False,
-                        verbose_name="ID",
-                    ),
-                ),
-                ("name", models.CharField(max_length=255, unique=True)),
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('show_time', models.DateTimeField()),
+                ('cinema_hall', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='movie_sessions', to='db.cinemahall')),
+                ('movie', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='movie_sessions', to='db.movie')),
             ],
         ),
     ]
```

Comment:
Do not modify the existing migration, add a new one with all required changes.

Reply:
How? I just delete couple of my migrations and then run migrations again, just to make it "initial" . I don`t get it where i modify my migration

Reply:
you don't need to delete the old migrations. it's ok if your new migration is not "initial", for new changes, create new migrations, but keep the old ones unchanged